documenters_aggregator/spiders/cceb.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import scrapy
+
+from datetime import datetime
+from pytz import timezone
+
+
+class CcebSpider(scrapy.Spider):
+    name = 'cceb'
+    allowed_domains = ['http://cookcountyclerk.com/elections/electoralboard/Pages/default.aspx']
```

Comment:
this should just be `cookcountyclerk.com`...

perhaps we should clarify this point in the docs / inline comments in the genspider templates?